### PR TITLE
Add help panels throughout app

### LINKED ISSUE #120: Document "multi sort" operations in LineUp
e.g., sort first by synthesis probability, then some other value.

### LINKED ISSUE #122: Improve UI labels and add infotags to app

### PR DESCRIPTION
**Pull request by waxlamp**
Closes #120 
Closes #122

### CODE REVIEW

**Comment by sgratzl** [comment]
can you fix the linting issues, please

File: src/components/graph-vis/helpPanel.js [hunk 1]
```diff
@@ -0,0 +1,320 @@
...
+  );
+}
+
+export function SearchHelp() {
+  return (
+    <HelpPanelLabel name='search-help' label='Search'>
+      <p>
+        Use this panel to search for items in the loaded dataset. Search results
+        will match the phrase used anywhere in the name of the item. For
+        example, searching for <tt>N</tt> will match Nitrogen (<tt>N</tt>), but
```
Comment (sgratzl) [inline src/components/graph-vis/helpPanel.js]: tt tag is obsolete: 

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/tt

Comment (sgratzl) [reply src/components/graph-vis/helpPanel.js]: what about: strong, i, pre, code?

File: src/components/graph-vis/helpPanel.js [hunk 2]
```diff
@@ -0,0 +1,320 @@
...
+
+      <p>
+        The pinned/neighbor/subpsace state of each material is also visible in
+        the item detail panel, which can be opened by clicking on an item, or
+        its pinned badge.
+      </p>
+
+      <p>
+        These states can also be toggled by interacting with the map directly:
+        Ctrl-click will pin an item; shift-click will pin it and show neighbors;
```
Comment (sgratzl) [inline src/components/graph-vis/helpPanel.js]: later on you wrapped `ctrl-click` with a tag. For consistency I would wrap all buttons and actions using e.g. a `<code>Ctrl-click</code>`, also like `"Start"` will become `<code>Start</code>`

File: src/components/graph-vis/helpPanel.js [hunk 3]
```diff
@@ -0,0 +1,320 @@
...
+        name={name}
+        children={children}
+      />
+    </span>
+  );
+}
+
+export function SearchHelp() {
+  return (
+    <HelpPanelLabel name='search-help' label='Search'>
```
Comment (sgratzl) [inline src/components/graph-vis/helpPanel.js]: note: most linters would enforce using `"` in jsx attributes

File: src/components/graph-vis/helpPanel.js (hunk 1 above)
Comment (waxlamp) [reply src/components/graph-vis/helpPanel.js]: See b81bef379e3906c69b45bd2149702c5459f7cbb6. How about `<samp>`? :smile_cat:

File: src/components/graph-vis/helpPanel.js (hunk 2 above)
Comment (waxlamp) [reply src/components/graph-vis/helpPanel.js]: See c616e0d8ed3811ea36a0ddc47706d450d5472463.

File: src/components/graph-vis/helpPanel.js (hunk 3 above)
Comment (waxlamp) [reply src/components/graph-vis/helpPanel.js]: We should definitely add linter support to this project, but for right now there's d706b2b29fc6b4d56a2981ad075ff069be354772.

**Pull request merged**